packages/openapi-framework/index.ts

```diff
@@ -227,6 +228,7 @@ export default class OpenAPIFramework implements IOpenAPIFramework {
       }
     });
     routes = routes.sort(byRoute);
+    this.logger.debug(`${this.loggingPrefix}routes=`, routes.map(r => r.path));
```

Comment:
i'm not crazy about this because `routes.map` is going to get executed even if debug logging isn't enabled.